docs/sources/administration/service-accounts/add-service-account-token.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Add a token to a service account in Grafana'
```

Comment:
Should add screenshots next to the instructions?

Reply:
https://github.com/grafana/grafana/pull/46801/files#r843259075